components/text-field/README.md

```diff
@@ -0,0 +1,140 @@
+<!--Copyright (c) Coherent Labs AD. All rights reserved. Licensed under the MIT License. See License.txt in the project root for license information. -->
+The gameface-text-field is part of the Gameface custom components suite. As most of the components in this suite, it uses slots to allow dynamic content.
```

Comment:
Because if it was input we need to support the other types like date, submit, color, etc.
With `gameface-text-field` we target all the input behaviors that are related just to typing text.

Reply:
It makes sense (what @alien1976 said) because we might add more components for color, date, etc. which are going to be separate for the moment as it is with the range slider.